hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyDeleteRequest.java

```diff
@@ -87,7 +89,10 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     OzoneManagerProtocolProtos.KeyArgs.Builder newKeyArgs =
         keyArgs.toBuilder().setModificationTime(Time.now()).setKeyName(keyPath);
 
+    long startNanosDeleteKeyResolveBucketAndCheckAclsLatency = Time.monotonicNowNanos();
     KeyArgs resolvedArgs = resolveBucketAndCheckAcls(ozoneManager, newKeyArgs);
```

Comment:
You can wrap the [`MetricUtil#captureLatencyNs`](https://github.com/apache/ozone/blob/master/hadoop-hdds/common/src/main/java/org/apache/hadoop/util/MetricUtil.java) block directly around the `resolveBucketAndCheckAcls` call.

Reply:
In  resolveBucketAndCheckAcls also we need to add a try and catch block to handle exception and then we can use captureLatencyNs there.